### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by dwram**
### Summary

- Flash notices were updated to appropriately and `dynamically` reference the time-out constant limiting the ability to update a given post.

- There was a bug breaking Travis CI whereas during testing and upon executing the application helper's reload_page method, ActionController did not know the '.JS' format to reload. This was fixed by rendering a format it _did_ know, `format.html` and redirecting to posts_path, which in the given context is the "show" root.

- Upon successful registration,  Users are assigned a session ID enabling them to redirect successfully to the (GET) '/posts' page.

- Destroy feature was not working due to the foreign key violation; UserPosts records must be deleted first _before_ the record their FK is depending on (row of a given post ID) is deleted. This has been fixed. 
 
- Implemented three column structure on posts page with outermost columns fixed to emulate Facebook.

- Upon successful registration Users are logged in.

- Spec test was failing CI due to placeholder values not being treated as a readable node element in a simple query. Amended to fix.

### CODE REVIEW

**Review by phump81 — APPROVED** [review]

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,20 +1,57 @@
...
   def index
     @posts = Post.order(created_at: :desc)
   end
 
+  def update
+    return if post_params['message'].empty?
+
+    @post = Post.find(params[:id])
+    if !@post.updatable? && (@post.user_id == current_user.id)
+      flash[:alert] = "#{@post.update_time} seconds have elapsed since the post was created. It can no longer be updated"
```
Comment (02ship) [inline app/controllers/posts_controller.rb:31]: Do we want to display this in seconds? A minor point, but I feel minutes makes more sense given our limit

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1,12 +1,60 @@
...
+            <%= form_with(model: post) do |form| %>
+              <%= form.text_area :message, id: 'update_content' %>
+              <br/>
+              <%= form.submit "Update" %>
+            <% end %>
+          </div>
+        <% end %>
+        <p> <%= post.created_at.localtime.strftime('%k:%M') %> | <%= post.created_at.localtime.strftime('%d/%m/%Y') %><p>
+        <% if post.user_id == current_user.id %>
+        <div id="elete_post">
```
Comment (ralphm10) [inline app/views/posts/index.html.erb:41]: should this be "delete post"?

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (dwram) [reply app/controllers/posts_controller.rb:31]: Will ticket

**Review by 02ship — APPROVED** [review]
Time limit to be resolved with new ticket, typo raised by Ralph resolved

**Pull request merged**